index.html

```diff
@@ -38,6 +39,203 @@ <h1 class="text1">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="snapshot-image">
+    <img src="images/tonic.png" alt="tonic-Page">
+</div>
+<div class="tonicmain">
+    <p>Tonic</p>
+</div>
+<div class="listitems">
+<ul>
+    <li><span>Canopy</span></li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>Back End Dev</li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>2015</li>
+</ul>
+</div>
+
+<div class="selection">
+    <p>A daily selection of 
+        privately personalized 
+        reads; no accounts or 
+        sign-ups required.</p>
+</div>
+<div class="languages">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="snapshot-image">
+    <img src="images/multypost.png" alt="">
+</div>
+<div class="tonicmain">
+    <p>Tonic</p>
+</div>
+<div class="listitems">
+<ul>
+    <li><span>CANOPY</span></li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>Back End Dev</li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>2015</li>
+</ul>
+</div>
+
+<div class="selection">
+    <p>A daily selection of 
+        privately personalized 
+        reads; no accounts or 
+        sign-ups required.</p>
+</div>
+<div class="languages">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="tonicmain">
+    <p>About Myself</p>
+</div>
+<div class="selection">
+    <P>Hello I’m a software developer! 
+        I can help you build a product , 
+        feature or website Look through 
+        some of my work and experience! 
+        If you like what you see and have
+         a project you need coded, don’t 
+         hestiate to contact me.</P>
+</div>
+<div class="languages">
+    <a href="#">LET'S CONNECT</a>
+</div>
+<div class="icons">
+    <ul>
+        <li><a href="https://twitter.com"><img src="images/twitter.png" alt="twitter"></a></li>
+        <li><a href="https://Linkedin.com"><img src="images/Linkedin icon.png" alt="Linkedin"></a></li>
+        <li><a href="https://medium.com"><img src="images/medium.png" alt="medium"></a></li>
+        <li><a href="https://github.com"><img src="images/github.png" alt="github"></a></li>
+        <li><a href="https://angellist.com"><img src="images/angellist.png" alt="angellist"></a></li>
+    </ul>   
+</div>
+<div class="button">
+    <button type="button">Get my resume</button>
+</div>
+</section>
+  
+</section>
+<section class="grid-container">
```

Comment:
- [ ] Kindly ensure that your list of technologies matches the Figma design in terms of color, font weight and font size 👍🏽

Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/192098897-cecb9fe0-6a90-42ed-b1b4-05df11addb57.png)
 
Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/192099024-34ea2a18-3fb8-4bd4-a749-262ece1c8f73.png)